external/test-runtime/XUnit.Runtime.depproj

```diff
@@ -51,7 +51,7 @@
       <Version>0.10.0-alpha-experimental</Version>
     </PackageReference>
     <PackageReference Condition="'$(TargetGroup)' != 'uapaot'" Include="Microsoft.DotNet.PlatformAbstractions">
-      <Version>$(MicrosoftDotNetPlatformAbstractionsPackageVersion)</Version>
+      <Version>2.0.4</Version>
```

Comment:
Would it make sense to continue hard-coding this in the dependencies.props file?  And just unhook the `XmlUpdateStep`?

Reply:
If we do that and keep the same name then source-build/prodcon dependency flow will update this version which we also want to avoid, so for now I'm just hard-coding it here.

Reply:
You are correct about prodcon but in source-build when you produce a tarball this property will end up getting bumped up to the latest when you build that tarball.